unistd/unistd.cpp

```diff
@@ -579,3 +579,11 @@ int vasprintf(char **ptr, const char *format, va_list arg)
 	*ptr = p;
 	return rv;
 }
+
+int asprintf(char **ret, const char *format, ...)
+{	va_list ap;
+	va_start(ap, format);
+	int retval = vasprintf(strp, format, ap);
+	va_end(ap);
+	return returnval;
```

Comment:
```suggestion
	return retval;
```
?

Is this still needed?

Reply:
It's not needed as I closed the PR this was needed for, but the change could still be useful.